src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const preparedContent = tabs.filter(tab => tab.id === activeTabId)[0].content;
```

Comment:
Filter is not the proper method here

If you want to get 1 element use `find`. Also, note that find can return `undefined`, so you should show the content of the first tab:


![image](https://github.com/mate-academy/react_tabs-js/assets/86923089/ccee74cf-dd4f-4a9b-a8a7-628e52bd04f9)

Reply:
done